src/utils/gameStateUtil.ts

```diff
@@ -33,7 +33,7 @@ export const gameStateUtil = {
       );
     }
 
-    const isSaved = databaseUtil.save(state);
+    const { isSaved } = databaseUtil.save(state);
```

Comment:
Should this be `{ isSaved, error }` here so we use `error` in the `DeveloperError` below?

Reply:
I didn't because `error` is `unknown` and you have to write  a bit of logic to make TypeScript happy with logging the error. I just made [this ticket](https://github.com/normal-devs/klicker-knight/projects/1#card-77267482) to handle that.

It's not critical now, because you will probably not get that error.